### PR TITLE
Update pathways support to reflect adopted spec

### PR DESCRIPTION
**Pull request by kurtraschke**
**Summary:**

Update pathways support to reflect google/transit#143.

**Expected behavior:** 

Support for pathways spec as adopted in google/transit#143, including pathways and levels.

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Run the unit tests with `mvn test` to make sure you didn't break anything
- [ ] Format the title like "Fix #<issue_number> - short description of fix and changes"
- [x] Linked all relevant issues

### CODE REVIEW

**Comment by kurtraschke** [comment]
Note that this is dependent on resolution of https://github.com/OneBusAway/onebusaway-csv-entities/pull/12 for correct functionality (so, in theory depends on a release of `onebusaway-csv-modules` and a version bump here).

**Comment by sdjacobs** [comment]
> Note that this is dependent on resolution of OneBusAway/onebusaway-csv-entities#12 for correct functionality.
Can you add a test for this? I think we should resolve the upstream issue first and ensure correct functionality before we merge.

**Comment by kurtraschke** [comment]
I've added a test for the `onebusaway-csv-modules` fix in https://github.com/OneBusAway/onebusaway-csv-entities/pull/12/commits/706189a35fd75ff2fc89f78159db5e7d9ba5a32e.

File: onebusaway-gtfs/src/main/java/org/onebusaway/gtfs/model/Level.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+@CsvFields(filename = "levels.txt", required = false, prefix = "level_")
+public class Level extends IdentityBean<AgencyAndId> {
+
+    private static final long serialVersionUID = 1L;
+
+    @CsvField(mapping=DefaultAgencyIdFieldMappingFactory.class)
+    private AgencyAndId id;
+    @CsvField
+    private double index;
```
[the remark this answers is not in the document]
Comment (sdjacobs) [reply onebusaway-gtfs/src/main/java/org/onebusaway/gtfs/model/Level.java]: (Though based on the one response to my comment, it probably won't change.)

File: onebusaway-gtfs/src/test/java/org/onebusaway/gtfs/serialization/GtfsReaderStopsTest.java [hunk 2]
```diff
@@ -41,6 +42,7 @@ public void setup() throws IOException {
   }
 
   @Test
+  @Ignore
```
[the remark this answers is not in the document]
Comment (kurtraschke) [reply onebusaway-gtfs/src/test/java/org/onebusaway/gtfs/serialization/GtfsReaderStopsTest.java]: Done in a61d6d7.

**Pull request merged**